src/components/video/VideoJS.jsx

```diff
@@ -11,11 +12,56 @@ window.videojs = videojs;
 // eslint-disable-next-line import/no-extraneous-dependencies
 require('videojs-vjstranscribe');
 
+function useTranscripts({ player, customOptions }) {
+  const shouldUseTranscripts = customOptions?.showTranscripts && customOptions?.transcriptUrls;
```

Comment:
@adamstankiewicz Are we expecting a boolean value here?
This statement does not explicitly return a boolean. Instead, it will return the value of transcriptUrls if it exists. 

We can do something like
`const shouldShowTranscripts = customOptions?.showTranscripts && Boolean(customOptions?.transcriptUrls);
`